lib/variant_data.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * This function takes a list of frequnce sources and returns a sorted list of sources.
+ * @param {Array} freqKeys - A list of frequency sources
+ * @returns A sorted list of frequency sources
+ */
+export function getSortedFreqKeys(freqKeys) {
+  const keyOrder = [
+    "bravo_af",
+    "gnomad_af_total",
+    "gnomad_af_afr",
+    "gnomad_af_afr_female",
+    "gnomad_af_afr_male",
+    "gnomad_af_ami",
+    "gnomad_af_ami_female",
+    "gnomad_af_ami_male",
+    "gnomad_af_amr",
+    "gnomad_af_amr_female",
+    "gnomad_af_amr_male",
+    "gnomad_af_asj",
+    "gnomad_af_asj_female",
+    "gnomad_af_asj_male",
+    "gnomad_af_eas",
+    "gnomad_af_eas_female",
+    "gnomad_af_eas_male",
+    "gnomad_af_female",
+    "gnomad_af_fin",
+    "gnomad_af_fin_female",
+    "gnomad_af_fin_male",
+    "gnomad_af_male",
+    "gnomad_af_nfe",
+    "gnomad_af_nfe_female",
+    "gnomad_af_nfe_male",
+    "gnomad_af_oth",
+    "gnomad_af_oth_female",
+    "gnomad_af_oth_male",
+    "gnomad_af_sas",
+    "gnomad_af_sas_male",
+    "gnomad_af_sas_female",
+    "gnomad_af_raw",
+  ];
```

Comment:
Could move `keyOrder` outside `getSortedFreqKeys()` so that it doesn’t get allocated and possibly garbage collected every time you call this function.

Reply:
Will do!